web/secondary_toolbar.js

```diff
@@ -129,6 +134,18 @@ class SecondaryToolbar {
         eventDetails: { mode: SpreadMode.EVEN },
         close: true,
       },
+      {
+        element: options.ltrReadingDirectionButton,
+        eventName: "changedirection",
```

Comment:
```suggestion
      {
        element: options.readingDirectionLtrButton,
        eventName: "switchreadingdirection",
```
(With *similar* changes below as well.)